addons/config/README.md

```diff
@@ -0,0 +1,33 @@
+# Addon Config CR templates
+
+This folder stores YTT template for generating default addon config CRs used by ClusterBootstrapTemplate. It will be invoked by the make targed at the root of tanzu-framework.
+
+The make target needs 4 inputs, which are GVK of the config CR, and the current TKR version.
+
+``` bash
+make generate-package-config apiGroup=cni.tanzu.vmware.com kind=AntreaConfig version=v1alpha1 tkr=v1.23.3---vmware.1-tkg.1
```

Comment:
Could we document namespace param as well?

Reply:
Documented